src/functions/postLogs/framework/handler.ts

```diff
@@ -36,7 +36,7 @@ export async function handler(
       message:
         "Bad Request: request body should contain JSON array of log messages.",
     },
-    400
+    400,
```

Comment:
Is it worth pulling `HttpStatus` into this repo and using it too?

Reply:
Yes, good shout. Looks like I'll then be able to delete a few files too.